packages/laconia-core/src/laconia.js

```diff
@@ -35,15 +35,37 @@ module.exports = app => {
     }
   };
 
+  const registerSingle = (name, factory, options) => {
+    checkFunction("register", factory);
```

Comment:
We should improve `checkFunction` with `name` in this case?

Reply:
Not sure what you mean...? The name would the factory instance name, where as the factory needs to be a function.

Reply:
if we got an [error](https://github.com/laconiajs/laconia/pull/284/files#diff-2f57d2e9b116f153115da98905ff6503R180) the message contains only `factory`, but `name` help identify where problem is, without stacktrace.